### ISSUE TITLE
"uninitialized constant Logger" error when using Google::Cloud::Logging::Logger

### ISSUE THREAD

**Issue by hxiong388** [GoogleCloudPlatform/google-cloud-ruby#1174]
When running the integration test on sinatra + GAE, the logging test throws this error:
```
/app/google-cloud-logging/lib/google/cloud/logging/logger.rb:147:in `initialize': uninitialized constant Logger (NameError)
 	from /app/google-cloud-logging/lib/google/cloud/logging/project.rb:382:in `new'
 	from /app/google-cloud-logging/lib/google/cloud/logging/project.rb:382:in `logger'
 	...
```

I think this exception is introduced in d5af77d9e6ed815fa4bd2feb01d9eed1125b05bd, where the code references "::Logger::Formatter" module without first `require "logger"`.

**Comment by blowmage** [comment]
Wasn't this already addressed by #1173?

**Comment by hxiong388** [comment]
Yeah, i didn't see #1173 was in already. Didn't expect y'all to work so hard on xmax eve.

closed via #1173
